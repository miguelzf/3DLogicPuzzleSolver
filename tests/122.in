- - - - -
- B R - -
- - - - -
- - - - -
- - - - Y

G - - - G
- - - - -
Y - R - B
- - - - -
- - - - -

L - - P -
- - - L -
- P - - -
- - - - -
- - - - -
