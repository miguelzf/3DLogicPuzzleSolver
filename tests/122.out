
Solution:
g  g  g  g  y  
g  B  R  g  y  
g  b  r  g  y  
g  b  r  g  y  
g  b  r  g  Y  
=============
G  b  r  g  G  ||  L  p  p  P  y  
-  b  r  -  -  ||  l  p  -  L  y  
Y  b  R  -  B  ||  l  P  -  l  y  
y  b  b  b  b  ||  l  l  l  l  y  
y  y  y  y  y  ||  y  y  y  y  y  
